src/pybind/mgr/dashboard/frontend/src/app/shared/components/custom-login-banner/custom-login-banner.component.html

```diff
@@ -0,0 +1,2 @@
+<p class="p-3 m-0 font-weight-bold"
+   *ngIf="customLoginBannerText">{{ customLoginBannerText }}</p>
```

Comment:
I see the text spans over the octopus background image, which might make it less readable. Should we impose some cropping/padding to avoid overlapping that region?

Reply:
Done.